src/FxCop/Desktop.Analyzers/CSharp/CSharpDesktopAnalyzers.csproj

```diff
@@ -100,8 +100,9 @@
   </ItemGroup>
   <ItemGroup />
   <ItemGroup>
+    <Compile Include="Security\DoNotUseInsecureCryptographicAlgorithms.cs" />
     <Compile Include="Shared\SyntaxNodeHelper.cs" />
-    <Compile Include="Security\DoNotCatchCorruptedStateExceptionsAnalyzer.cs" />
+    <Compile Include="Security\DoNotCatchCorruptedStateExceptions.cs" />
```

Comment:
Looks like this one and the one  above "DoNotUseInsecureCryptographicAlgorithms" both didn't use the class names as file names. I believe this should be fixed. 

@genlu 
What do you think?